test_utils/fake_enterprise_api.py

```diff
@@ -177,6 +177,18 @@ def mock_ent_courses_api_with_pagination(self, enterprise_uuid, course_run_ids):
                 content_type='application/json',
             )
 
+    def mock_ent_courses_api_with_error(self, enterprise_uuid):
```

Comment:
nit: `ent ~ enterprise`

Reply:
@asadiqbal08 the quality will fail due to long variable name:
```
quality runtests: commands[5] | pylint test_utils
************* Module test_utils.fake_enterprise_api
C:110, 4: Invalid method name "mock_enterprise_courses_api_with_pagination" (invalid-name)
```